brd-ios/breadwallet/src/FlowControllers/RecoveryKeyFlowController.swift

```diff
@@ -282,9 +282,11 @@ class RecoveryKeyFlowController {
         })
         
         pinViewController.transitioningDelegate = RecoveryKeyFlowController.pinTransitionDelegate
-        pinViewController.modalPresentationStyle = .overFullScreen
-        pinViewController.modalPresentationCapturesStatusBarAppearance = true
         
-        navigationController.present(pinViewController, animated: true, completion: nil)
+        let navigation = UINavigationController(rootViewController: pinViewController)
```

Comment:
would RootNavigationController work here?

Reply:
yes, it's working